assets/css/utils.sass

```diff
@@ -28,3 +44,40 @@
 .hidden
   display: none
   visibility: hidden
+
```

Comment:
i know the name may not be great, but i think of utils as generic useful things - I'm gonna move the .modal-dialog stuff to view.sass